src/app/asset-page/asset-page.component.ts

```diff
@@ -453,6 +455,27 @@ export class AssetPage implements OnInit, OnDestroy {
     this._group.hasPrivateGroups()
   } // OnInit
 
+  get showDownloadDropdown() {
+    const isLoggedIn = this.user && this.user.isLoggedIn;
+    const publicDownload = this.assets && this.assets[0] && this.assets[0].publicDownload;
+    const downloadLink = this.assets && this.assets[0] && this.assets[0].downloadLink;
+    const userStatus = this.user && this.user.status;
```

Comment:
The `userStatus` here is really an indicator if the user is "institutionally authed" right? Wondering if renaming it to reflect such would help the reading of the if conditions we have below.

Reply:
This was the value that Spencer from AUTH suggested, so I certainly hope so! 😅 It gave me what I expected when loading the site in various ways.

Reply:
Renamed the variable to "userAuthedWithInstitution" in ea5014d